cdc/redo/writer/file.go

```diff
@@ -255,13 +255,13 @@ func (w *Writer) Close() error {
 	}
 	w.Lock()
 	defer w.Unlock()
+	defer w.running.Store(false)
 
 	err := w.close()
 	if err != nil {
 		return err
 	}
```

Comment:
Can we add a unit test to check `w.close` returns nil error and `running` state is set properly.